diff --git a/node_modules/react-native-toast-message/src/index.js b/node_modules/react-native-toast-message/src/index.js
index 3a24b2f..1e0800c 100644
--- a/node_modules/react-native-toast-message/src/index.js
+++ b/node_modules/react-native-toast-message/src/index.js
@@ -121,6 +121,9 @@ class Toast extends Component {
       },
       onPanResponderRelease: (event, gesture) => {
         this._animateRelease(gesture);
+      },
+      onPanResponderEnd: (event, gesture) => {
+        this._animateRelease(gesture);
       }
     });
   }
@@ -194,7 +197,7 @@ class Toast extends Component {
       value = 1 - dy / 100;
     }
 
-    const treshold = 0.65;
+    const treshold = 0.99;
     if (value <= treshold || Math.abs(vy) >= treshold) {
       this.hide({
         speed: Math.abs(vy) * 3
