diff --git a/node_modules/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js b/node_modules/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js
index 7c794ea..f0584d0 100644
--- a/node_modules/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js
+++ b/node_modules/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js
@@ -26,6 +26,7 @@ import type {
 } from '../View/ViewPropTypes';
 import type {KeyboardEvent} from './Keyboard';
 
+
 type Props = $ReadOnly<{|
   ...ViewProps,
 
@@ -56,6 +57,22 @@ type State = {|
   bottom: number,
 |};
 
+function debounce(func, wait, immediate) {
+  console.log('debunce')
+  var timeout;
+  return function() {
+    var context = this, args = arguments;
+    var later = function() {
+      timeout = null;
+      if (!immediate) func.apply(context, args);
+    };
+    var callNow = immediate && !timeout;
+    clearTimeout(timeout);
+    timeout = setTimeout(later, wait);
+    if (callNow) func.apply(context, args);
+  };
+};
+
 /**
  * View that moves out of the way when the keyboard appears by automatically
  * adjusting its height, position, or bottom padding.
@@ -90,7 +107,7 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
     return Math.max(frame.y + frame.height - keyboardY, 0);
   }
 
-  _onKeyboardChange = (event: ?KeyboardEvent) => {
+  _onKeyboardChange = debounce((event: ?KeyboardEvent) => {
     if (event == null) {
       this.setState({bottom: 0});
       return;
@@ -102,6 +119,8 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
     if (this.state.bottom === height) {
       return;
     }
+    console.log({duration, easing, endCoordinates, height})
+
 
     if (duration && easing) {
       LayoutAnimation.configureNext({
@@ -114,7 +133,9 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
       });
     }
     this.setState({bottom: height});
-  };
+  }, 200);
+
+
 
   _onLayout = (event: ViewLayoutEvent) => {
     this._frame = event.nativeEvent.layout;
diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
new file mode 100644
index 0000000..361f5fb
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.env
@@ -0,0 +1 @@
+export RCT_METRO_PORT=8081
